src/physics/cam/nn_interface_cam.F90

```diff
@@ -545,7 +544,7 @@ subroutine CAM_var_conversion(qv, qc, qi, r, tabs, t)
         !!          Using a grid that does not match the SAM grid for input/output
         !!          data will produce unphysical values.
 
-        integer :: nx, nz
+        integer :: ncol, nz
```

Comment:
should nz also be renamed?

Reply:
You are right, `ncol` and `nz` are pretty clear without further explanation, both `pver` and `nlev` would need comments to tell people what they are supposed to be. `nplevel` might work? Or just stay with  `nz` for now.